real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/ui/Gui.java

```diff
@@ -27,6 +27,8 @@
 	private MainGuiWindow mainGuiWindow;
 
 	private static Gui instance;
+	private static RobobuggyJFrame rmw;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Possible unsafe assignment to a non-final static field in a constructor.](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2370173536/issues/source?bid=2938665&fileBranchId=3199438#l30)